lib/bloc/explorer/explorer_bloc.dart

```diff
@@ -350,6 +351,91 @@ class ExplorerBloc extends Bloc<ExplorerEvent, ExplorerState> {
     }
   }
 
+  Future<Account> _syncAccountUnstakes(Account account) async {
+    try {
+      // TODO(#542): use Unstakes instead of AddressBlocks
+      /// retrieve all Block Hashes
+      AddressBlocks? unstakes = (await explorer.address(
+              value: account.address,
+              // TODO(#542): get paginated Unstakes from unstake tab instead of AddressBlocks
+              tab: 'blocks') as PaginatedRequest<AddressBlocks?>)
+          .data;
+
+      if (unstakes != null) {
+        /// check if the list of transaction is already in the database
+        // TODO(#542): use .unstakes from instead of .blocks
+        for (int i = 0; i < unstakes.blocks.length; i++) {
+          String unstakeHash = unstakes.blocks[i].hash;
+          UnstakeEntry? unstakeEntry =
+              database.walletStorage.getUnstake(unstakeHash);
+          // TODO(#542): use UnstakeInfo from instead of BlocInfo
+          BlockInfo unstakeInfo = unstakes.blocks.elementAt(i);
+
+          if (unstakeEntry != null) {
+            /// this mintEntry.status check for "confirmed" is in the local database
+            if (unstakeEntry.status != TxStatusLabel.confirmed) {
+              UnstakeEntry unstakeEntry =
+                  await explorer.getUnstake(unstakeInfo);
+              await account.addUnstake(unstakeEntry);
+            }
+          } else {
+            UnstakeEntry unstakeEntry = await explorer.getUnstake(unstakeInfo);
+            await account.addUnstake(unstakeEntry);
+          }
+        }
+
+        account.unstakeHashes.clear();
+        account.unstakeHashes
+            .addAll(unstakes.blocks.map((unstake) => unstake.hash));
+      }
+      return account;
+    } catch (e) {
+      print('Error syncing mints $e');
+      rethrow;
+    }
+  }
+
+  Future<Account> _syncAccountStakes(Account account) async {
+    try {
+      // TODO(#542): use Stakes instead of AddressBlocks
+      /// retrieve all Block Hashes
+      AddressBlocks? stakes = (await explorer.address(
+              value: account.address,
+              // TODO(#542): get paginated Stakes from 'stake' tab instead of AddressBlocks
+              tab: 'blocks') as PaginatedRequest<AddressBlocks?>)
+          .data;
+
+      if (stakes != null) {
```

Comment:
`dart fix` detects this condition as unnecessary because `stakes` can't be `null`

Reply:
stakes can be null